src/python/platforms/android/adb.py

```diff
@@ -147,9 +167,32 @@ def run():
   return bytes_output.strip().decode('utf-8', errors='ignore')
 
 
+def copy_file(filename, src_dir, dest_dir, timeout=None):
```

Comment:
change the arguments to 

`copy_file_to_cuttlefish_host(src_path, dest_path, timeout=None)` here (where both are full paths).

Also, we only need to handle the cuttlefish case here right? This is called below from `recreate_gce_device`, which is only called on cuttelfish.

Reply:
My initial intent is to still keep the old logic that cuttlefish runs locally with clusterfuzz on host machine.
But we can add later if do need that case, removed for now.